packages/default/test/shared.js

```diff
@@ -112,8 +115,9 @@ export const sharedTestCases = (mode, torusSP, storageLayer, keyType) => {
         fail("key should be able to be reconstructed");
       }
     });
+
     it(`#should be able to reconstruct key when initializing with import key, manualSync=${mode}`, async function () {
-      const importedKey = new BN(generatePrivate(keyType));
+      const importedKey = keyType === "ed25519" ? await getRandomBytes(32) : generatePrivate(keyType);
```

Comment:
This line shows that our `generatePrivate` API might cause some confusion:

For `ed25519`, `generatePrivate ` does not generate a private key. (It generates a scalar over `edwards25519` curve.)

Not saying this is necessarily an issue, just wanted to point it out.

Reply:
Yeah, i understand your concern, its not the best way, But i think we will have to highlight this carefully in docs and examples.